setup.py

```diff
@@ -19,6 +19,7 @@
 
 install_requires = [
     "openai>=1.3",
+    "anthropic>=0.23.1",
```

Comment:
anthropic shouldn't be the required dependencies. it is optional, for the devs using anthropic models.

Reply:
instead you can update the `extra_kwargs` in the client.py and make an `anthropic_config` variable

Reply:
so, for autogen, other llm clients are just dev optional? not regular choices between openai and others?

Reply:
Hey @wenngong, just catching up on this one...

@Hk669 is right in that it's not a mandatory dependency for all AutoGen users to have anthropic's library installed. It is an optional install, e.g. `pip install pyautogen[anthropic]` which is defined in the "extra_require" section of the `setup.py`.

So we can leave the setup.py file as it was :)

Reply:
> Hey @wenngong, just catching up on this one...
> 
> @Hk669 is right in that it's not a mandatory dependency for all AutoGen users to have anthropic's library installed. It is an optional install, e.g. `pip install pyautogen[anthropic]` which is defined in the "extra_require" section of the `setup.py`.
> 
> So we can leave the setup.py file as it was :)

OK，got it, I will restore setup.py file. Thanks @Hk669 @marklysze